trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0..9]
```

Comment:
- Hey, to make this work correctly you're supposed to call the instance variable `@nameable`  instead of super().

`@nameable.correct_name[0..9]`